Governance_v_1_6 (#304)
MinBZK/Algoritmekader: docs/maatregelen/0-org-06-volwassenheidsmodel.md

---
title: Gebruik een AI-volwassenheidsmodel om te weten waar de organisatie staat

> id: urn:nl:ak:mtr:org:06

Reviewer: ```suggestion
id: urn:nl:ak:mtr:org-06
```

> toelichting: Pas een volwassenheidsmodel toe om te weten waar de organisatie staat op het gebied van AI

Reviewer: ```suggestion
toelichting: Breng de volwassenheid van je organisatie op het gebied van algoritmes en AI in kaart.
```

levenscyclus:
- organisatieverantwoordelijkheden
rollen:
- projectleider

> # - beleid-en-advies

Reviewer: ```suggestion
- beleid-en-advies
```

---

<!-- tags -->

## Maatregel

> Pas een volwassenheidsmodel toe om te weten waar jou organisatie staat op het gebied van AI

Reviewer: ```suggestion
Breng de volwassenheid van je organisatie op het gebied van algoritmes en AI in kaart.
```

## Toelichting 

> Pas een volwassenheidsmodel toe om te weten waar jou organisatie staat qua AI

Reviewer: ```suggestion
```

Om tot een passende algoritmegovernance voor een organisatie te komen, moet eerst worden vastgesteld wat op dit moment al is ingericht op het gebied van AI en algoritmes. 

> Op basis hiervan kunnen vervolgstappen worden gedefinieerd waar een organisatie aan kan gaan werken. Ook kunnen deze uitkomsten helpen bewustzijn over de uitdagingen te vergroten.

Reviewer: ```suggestion
Op basis hiervan kunnen vervolgstappen worden gedefinieerd zodat je met een handelingsperspectief hebt om je organisatie te verbeteren. Ook kunnen deze uitkomsten helpen bewustzijn over de uitdagingen te vergroten.
```

> Er zijn verschillende modellen om het volwassenheidsniveau te bepalen. Hieronder staan enkele voorbeelden. Deze modellen richten zich niet altijd specifiek op het beheer van algoritmes, maar kijken breder naar algoritmes in de organisatie of naar "AI-ethiek". Governance is altijd een onderdeel van deze modellen.

Reviewer: ```suggestion
```

Reviewer: dit verplaatst naar beneden

> ## Voorbeelden van volwassenheidsmodellen

Reviewer: ```suggestion
## Voorbeelden van volwassenheidsmodellen

Er zijn verschillende modellen om het volwassenheidsniveau te bepalen. Deze modellen richten zich niet altijd specifiek op het beheer van algoritmes, maar kijken breder naar algoritmes in de organisatie of naar "AI-ethiek". Governance is altijd een onderdeel van deze modellen.
```

### AI ethics maturity model van Krijger, Thuis, de Ruiter, Ligthart & Broekman (2023)

> Het [AI ethics maturity model van Krijger, Thuis, de Ruiter, Ligthart & Broekman (2023)](https://link.springer.com/article/10.1007/s43681-022-00228-7) is op basis van zes categorieën (awareness & culture, Policy, Governance, Communication & Training, Development Proces, en Tooling) op verschillende niveau’s. 

Reviewer: ```suggestion
Het [AI ethics maturity model van Krijger, Thuis, de Ruiter, Ligthart & Broekman (2023)](https://link.springer.com/article/10.1007/s43681-022-00228-7) brengt de volwassenheid op verschillende niveau's in kaart op basis van zes categorieën:

- awareness & culture, 
- policy
- governance
- communication & training
- pevelopment proces
- tooling
```

> ![Krijger, J., Thuis, T., de Ruiter, M. et al. The AI ethics maturity model: a holistic approach to advancing ethical data science in organizations. AI Ethics 3, 355–367 (2023)](https://github.com/user-attachments/assets/07860cdc-9d6c-46f4-aace-cfcdf71e114d)

Reviewer: ```suggestion
```

#### Het Datavolwassenheidsmodel van de IBDS

> Het datavolwassenheidmodel van de Interbestuurlijke Datastrategie (IBDS) kijkt naar hoe goed organisaties omgaan met data. Dit model is ook belangrijk voor kunstmatige intelligentie (AI), omdat data daar een grote rol speelt. Het model heeft een [beslishulp datavolwassenheid](https://realisatieibds.nl/groups/view/c23ab74c-adb4-424e-917d-773a37968efe/kenniscentrum-van-de-ibds/wiki/view/2447d2a8-6c48-468d-9739-00772688853f/beslishulp-datavolwassenheid) waarmee bepaald kan worden hoe volwassen een organisatie is met data. IBDS biedt ook een [gids](https://realisatieibds.nl/page/view/ad94d97c-4d48-443c-aedd-235b2d0ca8b6/wegwijzer-volwassenheidsmodellen) met verschillende manieren om dit te meten. 

Reviewer: ```suggestion
Het datavolwassenheidmodel van de Interbestuurlijke Datastrategie (IBDS) kijkt naar hoe goed organisaties omgaan met data. Dit model is ook belangrijk voor kunstmatige intelligentie (AI), omdat data daar een grote rol speelt. Het model heeft een [beslishulp datavolwassenheid](https://realisatieibds.nl/groups/view/c23ab74c-adb4-424e-917d-773a37968efe/kenniscentrum-van-de-ibds/wiki/view/2447d2a8-6c48-468d-9739-00772688853f/beslishulp-datavolwassenheid) waarmee bepaald kan worden hoe volwassen een organisatie is met data. IBDS biedt ook een [gids](https://realisatieibds.nl/page/view/ad94d97c-4d48-443c-aedd-235b2d0ca8b6/wegwijzer-volwassenheidsmodellen) met verschillende manieren om de datavolwassenheid in kaart te brengen. 
```

## Innovatie en volwassenheid

> Het gebruik van AI wordt vaak gezien als een vorm van innovatie. Hoe kan AI helpen bij het ondersteunen van nieuwe ideeën en het vinden van nieuwe toepassingen? Ook hiervoor zijn er ook modellen om de volwassenheid van innovatie te meten. Een voorbeeld hiervan is de [Innovatie Maturity Scan](https://www.rijksorganisatieodi.nl/innoveren-met-impact/onze-services/innovatie-maturity-scan) van Innoveren met Impact.

Reviewer: ```suggestion
Het gebruik van AI wordt vaak gezien als een vorm van innovatie. Hoe kan AI helpen bij het ondersteunen van nieuwe ideeën en het vinden van nieuwe toepassingen? Een voorbeeld van een volwassehnheidsmodel over innocatie is de [Innovatie Maturity Scan](https://www.rijksorganisatieodi.nl/innoveren-met-impact/onze-services/innovatie-maturity-scan) van Innoveren met Impact.
```

Het [AI ethics maturity model van Krijger, Thuis, de Ruiter, Ligthart & Broekman (2023)](https://link.springer.com/article/10.1007/s43681-022-00228-7) brengt de volwassenheid op verschillende niveau's in kaart op basis van zes categorieën:

- awareness & culture, 
- policy
- governance
- communication & training
- pevelopment proces
- tooling

### Het Datavolwassenheidsmodel van de IBDS
Het datavolwassenheidmodel van de Interbestuurlijke Datastrategie (IBDS) kijkt naar hoe goed organisaties omgaan met data. Dit model is ook belangrijk voor kunstmatige intelligentie (AI), omdat data daar een grote rol speelt. Het model heeft een [beslishulp datavolwassenheid](https://realisatieibds.nl/groups/view/c23ab74c-adb4-424e-917d-773a37968efe/kenniscentrum-van-de-ibds/wiki/view/2447d2a8-6c48-468d-9739-00772688853f/beslishulp-datavolwassenheid) waarmee bepaald kan worden hoe volwassen een organisatie is met data. IBDS biedt ook een [gids](https://realisatieibds.nl/page/view/ad94d97c-4d48-443c-aedd-235b2d0ca8b6/wegwijzer-volwassenheidsmodellen) met verschillende manieren om de datavolwassenheid in kaart te brengen. 

## Innovatie en volwassenheid

> Het gebruik van AI wordt vaak gezien als een vorm van innovatie. Hoe kan AI helpen bij het ondersteunen van nieuwe ideeën en het vinden van nieuwe toepassingen? Een voorbeeld van een volwassehnheidsmodel over innocatie is de [Innovatie Maturity Scan](https://www.rijksorganisatieodi.nl/innoveren-met-impact/onze-services/innovatie-maturity-scan) van Innoveren met Impact.

Reviewer 2: ```suggestion
Het gebruik van algoritmes wordt vaak gezien als een vorm van innovatie. Hoe kan AI helpen bij het ondersteunen van nieuwe ideeën en het vinden van nieuwe toepassingen? Een voorbeeld van een volwassenheidsmodel over innovatie is de [Innovatie Maturity Scan](https://www.rijksorganisatieodi.nl/innoveren-met-impact/onze-services/innovatie-maturity-scan) van Innoveren met Impact.
```

 
## Verdere stappen
Een analyse van volwassenheidsniveau kan helpen input te geven waar eerst op te richten.
Het is denkbaar dat het realiseren van algoritmegovernance vraagt om een organisatieverandering. 
Het is aan te raden om verantwoordelijkheden te beleggen voor het realiseren van algoritmegovernance.
De noodzaak voor implementatie van de AI-Verordening kan hier een katalysator voor zijn.  
Bewustzijn over de noodzaak voor algoritmegovernance moet op bestuurlijk niveau leven. 

## Aandachts- en actiepunten:
* Breng in kaart wat er al is aan (IT-)governance met één van de volwassenheidsmodellen.

> * In hoeverre kan hierbij worden aangesloten en zijn er zaken die aanvullend moeten worden ingericht specifiek voor een verantwoorde ontwikkeling en gebruik van algoritmes. 

Reviewer 2: ```suggestion
```
